### ISSUE TITLE
feature_detector: TypeError: 'numpy.float32' object is not iterable

### ISSUE THREAD

**Issue by JereMalinen** [thumbor/thumbor#910]
Certain image causes error in feature detector:

```
2017-04-18 14:02:05 tornado.application:ERROR Future exception was never retrieved: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/tornado/gen.py", line 1063, in run
    yielded = self.gen.throw(*exc_info)
  File "/usr/local/lib/python2.7/site-packages/thumbor/transformer.py", line 152, in smart_detect
    yield self.do_smart_detection()
  File "/usr/local/lib/python2.7/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/usr/local/lib/python2.7/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "/usr/local/lib/python2.7/site-packages/tornado/gen.py", line 1069, in run
    yielded = self.gen.send(value)
  File "/usr/local/lib/python2.7/site-packages/thumbor/transformer.py", line 180, in do_smart_detection
    detectors[0](self.context, index=0, detectors=detectors).detect(self.after_smart_detect)
  File "/usr/local/lib/python2.7/site-packages/thumbor/detectors/feature_detector/__init__.py", line 43, in detect
    for x, y in points.squeeze():
TypeError: 'numpy.float32' object is not iterable
 ```

I have tried version 6.1.3 and 6.3.0 with docker https://hub.docker.com/r/apsl/thumbor/

### To reproduce

- `docker run -e "DETECTORS=['thumbor.detectors.feature_detector']" -p 8001:8000 apsl/thumbor:6.3.0`
- Open http://localhost:8001/unsafe/300x300/smart/http%3A//s3-eu-west-1.amazonaws.com/testikala/thumbor_feature_detector_error.jpg

(works without the detector: `docker run -p 8001:8000 apsl/thumbor:6.3.0`)

**Comment by thor27** [comment]
Hi,

I figured out the issue. Can I use this image "http://s3-eu-west-1.amazonaws.com/testikala/thumbor_feature_detector_error.jpg" for the test case, and put it inside thumbor project?

**Comment by caeugusmao** [comment]
Fixed in this PR #1095
